### PR TITLE
Clearer compatibility matrices and Search series descriptions

### PR DESCRIPTION
**Pull request by yrodiere**
For ORM, before:

image

After:

image

For Search, before:

image

After:

image

For Validator, before:

image

After:

image

Other projects are mostly unaffected.

### CODE REVIEW

File: _data/projects/search/releases/7.2/series.yml [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-  document tree projection for the Lucene backend,
-  new APIs to work with analyzers,
-  compatibility with Elasticsearch 8.14 and 8.15,
-  compatibility with OpenSearch 2.14, 2.15 and 2.16,
-  upgrade to Lucene 9.11,
-  upgrade to Hibernate ORM 6.6,
-  other bugfixes, improvements and upgrades
+  Re-licensing as Apache License 2.0,
+  query string predicates on numeric/date fields,
+  analyzer API
```
Comment (marko-bekhta) [inline _data/projects/search/releases/7.2/series.yml:4]: Nobody reads these long summaries 😭 😆 👍🏻

File: _config/site.yml [hunk 2]
```diff
@@ -391,16 +397,16 @@ projects:
...
+      java_ee_bv:
         name: Bean Validation
         url: https://beanvalidation.org/
-      ee:
+      java_ee:
         name: Java EE
         url: https://www.oracle.com/java/technologies/java-ee-glance.html
-      jbv:
+      jakarta_ee_bv:
         name: Jakarta Bean Validation
```
Comment (marko-bekhta) [inline _config/site.yml:407]: What name should we use for this one ... since it got shorter with 3.1 (Jakarta Validation) ?

**Review by marko-bekhta — APPROVED** [review]
Looks nice 😃 🎉

File: _config/site.yml (hunk 2 above)
Comment (yrodiere) [reply _config/site.yml:407]: I'll let you do that update, as I suspect it's not _just_ about this entry -- there are likely other places in the website where we use "Jakarta Bean Validation".

+1 for "Jakarta Validation", even for early "Jakarta Bean Validation" releases, even if that's not 100% correct.

**Pull request merged**